src/fsharp/FSharp.Core/Linq.fs

```diff
@@ -782,7 +782,7 @@ module LeafExpressionConverter =
     // provides no other way to evaluate the expression.
     //
     // REVIEW: It is possible it is just better to interpret the expression in many common cases, e.g. property-gets, values etc.
-    let EvaluateQuotation (e: Microsoft.FSharp.Quotations.Expr) = 
+    let EvaluateQuotation (e: Microsoft.FSharp.Quotations.Expr) : obj = 
```

Comment:
you should download fsharp/fsharp#385 and do a file compare of `src` directory.
These changes are to make easier to diff codebase